### PR TITLE
Create moonVault.scss

### PR DESCRIPTION
**Pull request by Loesil**
Added new custom style for moonVault

#### Hi! What is your PR about?

- [ ] Add or edit a skin

### CODE REVIEW

File: skins/moonVault.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.moonVault {
```
Comment (gokaiorg) [inline skins/moonVault.scss:1]: Please use only lowercase for your file name and class selector. `moonvault.scss`
```suggestion
.moonvault {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The colours used in your skin make the submit button unreadable, please change with another.
Capture d’écran 2021-07-14 à 09 48 27

**Comment by Loesil** [comment]
link color changed to white

**Comment by Loesil** [comment]
Also changed name to lower case, like suggested

**Review by gokaiorg — CHANGES_REQUESTED** [review]
You still have an uppercase in the file name.

**Comment by Loesil** [comment]
resolved file name issue

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**